packages/cli/main.js

```diff
@@ -4,7 +4,10 @@ import hrp from "http-request-plus";
 import { format, parse } from "json-rpc-protocol";
 
 async function main() {
-  const params = require("minimist")(process.argv.slice(2));
+  const getopts = require("getopts");
+  const params = getopts(process.argv.slice(2), {
+    string: ["id"],
```

Comment:
Are you sure this is necessary?

The CLI should not have any knowledge regarding methods and their params.

Reply:
IMHO, it should be parsed in the method.

Reply:
IMHO, we should let minimist/getopts parses the params but we need to work-around the duck-typing behavior and manually convert all parsed params to strings.